packages/ra-ui-materialui/src/input/ResettableTextField.tsx

```diff
@@ -166,7 +166,6 @@ export const ResettableTextField = forwardRef(
                 variant={variant}
                 margin={margin}
                 className={className}
-                size="small"
```

Comment:
This is a BC. Can you extract the small prop and set its default value to `"small"` then pass it here? That should solve the issue as well

Reply:
"BC"? Sorry I don't follow.

But this wouldn't resolve the issue.
1. The prop is already passed with the "rest" props that follow immediately
2. Setting the default to "small" in the component would prevent it from being overrideable by the theme (RA's default theme already sets this default to "small", so we don't need to define it at a component level)

Reply:
I forgot about the theme default. Thanks for the reminder :) LGTM then but I'll wait for other core members opinion